File: app/src/main/java/org/ole/planet/myplanet/model/DocumentResponse.java
package org.ole.planet.myplanet.model;

import java.util.List;

public class DocumentResponse {
    private String total_rows;

    private String offset;

    private List<Rows> rows;

    public String getTotal_rows() {
        return total_rows;
    }

    public void setTotal_rows(String total_rows) {
        this.total_rows = total_rows;
    }

    public String getOffset() {
        return offset;
    }

    public void setOffset(String offset) {
        this.offset = offset;
    }

    public List<Rows> getRows() {
        return rows;
    }

    public void setRows(List<Rows> rows) {
        this.rows = rows;
    }

    @Override
    public String toString() {
        return "ClassPojo [total_rows = " + total_rows + ", offset = " + offset + ", rows = " + rows + "]";
    }
}

File: app/src/main/java/org/ole/planet/myplanet/model/DocumentResponse.kt
package org.ole.planet.myplanet.model

import kotlinx.serialization.Serializable

@Serializable
class DocumentResponse {
    private var totalRows: String? = null
    private var offset: String? = null
    var rows: List<Rows>? = null
    override fun toString(): String {
        return "ClassPojo [total_rows = $totalRows, offset = $offset, rows = $rows]"
    }
}
